app/models/activity.js

```diff
@@ -1,16 +1,19 @@
-import Model, {
+import {
   attr, belongsTo, hasMany
 } from '@ember-data/model';
-
+import ModelWithModifier from 'fe-redpencil/models/model-with-modifier';
 import CONFIG from 'fe-redpencil/utils/config';
 
-export default class Activity extends Model {
+export default class Activity extends ModelWithModifier {
   // Attributes.
   @attr('datetime') startDate;
   @attr('datetime') endDate;
   @attr('datetime') finalTranslationDate;
   @attr('string') name;
   @attr('string') mailContent;
+  @attr('datetime') created;
+  @attr('datetime') modified;
+  @attr('user') modifiedBy;
```

Comment:
Is geen attr maar een belongsTo
Maar dit moet hier niet staan, die belongsTo is al gedefinieerd in ModelWithModifier. dus gewoon weghalen